### PR TITLE
remove noarch and update tests

### PR DESCRIPTION
**Pull request by carlodri**

### CODE REVIEW

**Comment by carlodri** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -12,7 +12,6 @@ source:
   sha256: {{ sha256 }}
 
 build:
-  noarch: python
   number: 0
```
Comment (jakirkham) [inline recipe/meta.yaml]: Would bump build number in this case.

Comment (carlodri) [reply recipe/meta.yaml]: yep...sorry... too much multitasking!

Comment (jakirkham) [reply recipe/meta.yaml]: No worries. :)

**Pull request closed without merging**